### ISSUE TITLE
Chromatic snapshots are unstable

### ISSUE THREAD

**Issue by charpeni** [recharts/recharts#3926]
Recharts currently uses [Chromatic](https://www.chromatic.com/) to take snapshots of charts and prevent regressions. Unfortunately, somehow charts aren't stable and will be a little bit "jumpy".

We do have the same issue within our project using Recharts, we used [`diffThreshold`](https://www.chromatic.com/docs/threshold/), but it ended up being a big threshold and it doesn't flag regressions anymore.

Example: https://www.chromatic.com/test?appId=63da8268a0da9970db6992aa&id=6541f2c858c9611572300632.

<img width="2232" alt="Screenshot 2023-11-01 at 3 33 15 PM" src="https://github.com/recharts/recharts/assets/7189823/aac6e823-fbcb-4a98-bd25-025a51209a7c">

**Comment by ckifer** [comment]
Hmm, we've definitely caught regressions with it

**Comment by charpeni** [comment]
Ah, interesting!

I saw we already had a delay set for Chromatic: https://github.com/recharts/recharts/blame/b60424fefc6fae8d42cd8e9f040d26d38a1f89e6/storybook/preview.ts#L25.

I reused precisely the same value in our project, and that may be the issue. I'll try bumping the delay to something bigger and will report back in https://github.com/recharts/recharts/issues/3855.

Closing this one as this is directly a duplicate of #3855.
